sections/method/preprocessing.tex

```diff
@@ -1,51 +1,142 @@
+	\label{prepro:example3}
+\end{figure}
+
+
+\paragraph{Words Database Filtering}
+Due to a lack of danish POS-tagging resources, we instead check up the words in our document set against two databases containing only words of specific word classes, namely: nouns, adjectives, verbs, adverbs, and proper nouns.
+In practice this has the same effect as POS-tagging however words are not marked with their words class, instead they are removed if they do not appear in at least of the databases (containing only words of approved word classes).
+The two databases used are:  Wiktionary\footnote{https://github.com/fnielsen/awesome-danish (specifically Danish resources: https://people.compute.dtu.dk/faan/ps/Nielsen2016Danish.pdf)} and DanNet\cite{Pedersen2009DanNetTC}. 
 
-\paragraph{Stopwords and Word Frequency Filtering}
-Stopwords are removed, using \emph{nltk.corpus.stopwords}. 
-Words are removed if they appear fewer than 25 times in the document set or if they are used in over 25\% of the documents.\vejleder[inline]{why 25 and 20\%}
+Most words remaining at this step are contained within the databases, so this step does not remove as many words, however, this is mostly due to the step being taken late, due to having a longer computation time per word.
+This step mostly serves to remove names, and words from different languages, namely English.
+However, does also cut the odd non-word (eg. 'gimi') and abbreviations (eg. 'DK') that still exists at this point in the preprocessing.
+The example used in \autoref{prepro:example3} does not change at this step. 
+Instead, \autoref{prepro:example4} shows an example of another document that does change during this step.
 
-\paragraph{Words Database Filtering / POS-Tagging}
-Words are checked against the two word databases: Wiktionary and DanNet\vejleder[inline]{ref}. 
-The databases contain only: nouns, adjectives, verbs, adverbs, and proper nouns. 
-Words that are not contained within either database are removed.
+\begin{figure}[h]
+	\begin{quote}
+		\textit{
+			'\colorbox{pink}{country} kræver sommerens by \colorbox{pink}{night} sluttede afslappet band'
+		}
+	\end{quote}
+	\caption{Snippet of a document which had words removed during the word database filtering.}
+	\label{prepro:example4}
+\end{figure}
 
-\paragraph{Stemming}
-Words are reduced to their stems using \texttt{nltk.stem.snowball.DanishStemmer}. 
-This is done to combine word declensions, which reduces the number of words without removing too much information.
-Words that are stemmed into stopwords are also removed at this time.
+\paragraph{Lemmatization}
+Words are reduced to their stems using the python package \texttt{lemmy}\footnote{\url{https://pypi.org/project/lemmy/}}. 
+This is done by combining word inflections.
+However since we do not have proper POS-tagging, some words have multiple different meaning depending on the word class. In these cases we keep all inflected words.
```

Comment:
```suggestion
However since we do not have proper POS-tagging, some words have multiple meanings depending on the word class.
In these cases we keep all inflected words.
```